pkg/nodebootstrap/managed_windowss_test.go

```diff
@@ -0,0 +1,123 @@
+package nodebootstrap_test
+
+import (
+	"encoding/base64"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/nodebootstrap"
+	"strings"
+)
+
+var _ = Describe("Managed Windows UserData", func() {
+
+	type windowsEntry struct {
+		updateNodeGroup func(*api.ManagedNodeGroup)
+
+		expectedUserData string
+	}
+
+	DescribeTable("Managed Windows bootstrap", func(e windowsEntry) {
+		clusterConfig := api.NewClusterConfig()
+		clusterConfig.Metadata.Name = "windohs"
+		clusterConfig.Status = &api.ClusterStatus{
+			Endpoint:                 "https://test.com",
+			CertificateAuthorityData: []byte("test"),
+		}
+
+		ng := api.NewManagedNodeGroup()
+		ng.AMIFamily = api.NodeImageFamilyWindowsServer2019CoreContainer
+		//api.SetManagedNodeGroupDefaults(ng, clusterConfig.Metadata, false)
```

Comment:
Did you mean to comment this?

Reply:
Yes I did, I should've removed it before the PR.

Reply:
Also I wanted to ask for your suggestion on this file name. As you can see, I named the file managed_windowss_test.go. Without the extra 's' at the end of 'windowss' the file was not picked up by the compiler due to OS mismatch. Is there other way to get around this? Or is this file name an acceptable name?

Reply:
I did notice that. I think the current name has potential to be misunderstood as a typo and someone might rename it in the future. For now, we can work around this by renaming it to `windows_managed_test.go`. That way, the filename wouldn't match the `*_GOOS` build constraint. What do you think?

Reply:
That works! Thanks for the suggestion.